binder/Binder.cs

```diff
@@ -64,45 +72,60 @@ static void ParseCommandLineArgs(string[] args)
 
         static GeneratorKind ConvertToGeneratorKind(string gen)
         {
-            switch(gen.ToLower())
+            switch (gen.ToLower())

```

Comment:
Can you leave these formatting changes out of this commit? We can apply them afterwards in different commit as they are unrelated.